src/duke/task/Deadline.java

```diff
@@ -0,0 +1,17 @@
+package duke.task;
+
+public class Deadline extends Task {
+
+    public Deadline(String description,String dateTime) {
+        super(description);
+        if(!dateTime.isEmpty()) {
+            this.dateTime=dateTime.replaceFirst(" ",": ");
+        }
+        this.type = 'D';
```

Comment:
Might want to use constants or enums instead of string literals

Reply:
fixed in latest commit.